src/ghostwriter/hooks/ensure_lab.py

```diff
@@ -0,0 +1,50 @@
+"""Ensure that the user has a running lab."""
+
+import asyncio
+
+import structlog
+from rsp_jupyter_client.models.image import (
+    NubladoImageByClass,
+    NubladoImageClass,
+    NubladoImageSize,
+)
+
+from ..models.substitution import Parameters
+
+SPAWN_TIMEOUT = 90
```

Comment:
I wasn't sure.  In my head the hooks are self-contained (you'll notice we're using get_logger() to get loggers for them, for instance, rather than having it be passed somehow).  I feel like the scope of that timeout definition is only this particular hook.